### ISSUE TITLE
Window is located by mouse position, not screen argument

### ISSUE THREAD

**Issue by benoitvalery** [pyglet/pyglet#369]
Hi,
I'm using pyglet 1.5.15 on Linux debian 10.9 (x86-64), with a two-screen configuration. I would like to tell pyglet to display Window in a specific screen. So I'm just doing

```python
from pyglet.canvas import *
display = get_display()

class Software:
    def __init__(self):        
        self.window = Window(visible=False, fullscreen=True, screen=display.get_screens()[1])
        self.scenario = Scenario('Scenarios/scenario_test.txt')
        self.scheduler = Scheduler(self.scenario, self.window)
        
    def run(self):
        self.window.set_visible(True)
        self.scheduler.run()

if __name__ == '__main__':
    app = OpenMATB()
    app.run()
```
However, my Window gets displayed systematically on the screen where was my mouse at the moment I ran the program. On the other hand, providing screen information does correctly inform the Window instance about its size. Unfortunately, if my mouse is not located at the good screen, screen size information conflicts with its actual size, which is normal.

Is there any way to force pyglet to display Window at the specified screen, rather than at mouse location ?
Many thanks,

Benoît

**Comment by benmoran56** [comment]
Yes, this is expected behavior. The `screen` argument only works when you are using Fullscreen mode. For non-fullscreen Windows, the Desktop Environment will decide where it wants to initially place the Window. (often, by your mouse position).

You can, however, use the `Window.set_location(x, y)` method to reposition the Window later. 
If you inspect the `screens`, you can determine from their x, y, width, height values.  

My math could be off, but the following example should center the Window  in one of the displays: 

```py
import pyglet


window = pyglet.window.Window()
screen = pyglet.canvas.get_display().get_screens()[1]  # <-- my second display

target_x = (screen.x + screen.width / 2) - window.width / 2
target_y = (screen.y + screen.height / 2) - window.height / 2

window.switch_to()        # The Window must be active before setting the location
window.set_location(int(target_x), int(target_y))


pyglet.app.run()
```

**Comment by benoitvalery** [comment]
Ah, thanks, it works.
At the beginning, it was generating weird visual effects, because I was also resizing the Window with the `set_size` method. So I had to inform Window of its size at its initialization, and inform about its location following your suggestion.

It gives me the following code : 
```python
screen_index = 0  # or 1
screen=get_display().get_screens()[screen_index]

self.window = Window(caption=f'OpenMATB v{self.version}', width=screen.width,
                                     height=screen.height)
        
target_x = (screen.x + screen.width / 2) - self.window.width / 2
target_y = (screen.y + screen.height / 2) - self.window.height / 2
self.window.switch_to()        # The Window must be active before setting the location
self.window.set_location(int(target_x), int(target_y))
```
Your math was good, it works very well. Many thanks !

One last question. In my first attempt, I was setting the Window as `fullscreen`. You said that the `screen` argument worked for fullscreen configuration, which was my case (or at least my argument was True). Is it that in multiple screen configuration, we should not rely on this argument (`fullscreen`) and setting things more manually ? I mean : using your code, I ended with a fullscreen and correctly placed window, without having stipulated that I wanted a fullscreen anywhere, just inform about correct size and location.

Thanks for your responses !
